### DISCUSSION TITLE
How do I run different rtmo models posted in model zoo?

### DISCUSSION THREAD

**Discussion by jayavanth** [open-mmlab/mmpose#2950] [General] [answered]
I'm trying to run the models like this

`python demo/inferencer_demo.py video.mp4 --pose2d rtmo --vis-out-dir vis_results`

but the inference time is too slow on my GPU so I wanted to try `rtmo-s` model. How do I do that with inferencer (or any other tool)?

I tried running `python demo/inferencer_demo.py video.mp4 --pose2d rtmo-s --vis-out-dir vis_results` but that doesn't seem to work

Thank you!

**Comment by Ben-Louis** [comment, marked as answer]
Hi, you can refer to [this document](https://mmpose.readthedocs.io/en/latest/user_guides/inference.html#custom-pose-estimation-models) to learn how to use different models with inferencer

**Comment by jayavanth** [comment]
Thank you! This worked
`python demo/inferencer_demo.py video.mp4 --pose2d configs/body_2d_keypoint/rtmo/coco/rtmo-s_8xb32-600e_coco-640x640.py --pose2d-weights https://download.openmmlab.com/mmpose/v1/projects/rtmo/rtmo-s_8xb32-600e_coco-640x640-8db55a59_20231211.pth --vis-out-dir vis_results`
